lib/watcher/heart-beat.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+exports.worker = function(handle) {
+  var _heartBeatInterval = 15 * 1000;
```

Comment:
@bhajian could you please make this configurable and set the interval to something like `50ms` in your unit-tests? At the moment, `npm test` is taking 15 seconds to complete, which is driving me crazy.

/cc @sam-github